mayastor/src/bdev/nexus/nexus_state.rs

```diff
@@ -0,0 +1,36 @@
+use serde::{Deserialize, Serialize};
```

Comment:
nexus_state to me means something radically different then Config versions.

Reply:
Let us not start out with different versions. For testing, if you want to test those, then thats great and by all means but lets not start with deadcode from day zero by introductuuinng versions straight away.